<ui version="4.0" >
 <class>MagnifierWindow</class>
 <widget class="QMainWindow" name="MagnifierWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PPI Magnifier</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView_" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionViewZoomIn_" />
    <addaction name="actionViewZoomOut_" />
    <addaction name="actionViewPanLeft_" />
    <addaction name="actionViewPanRight_" />
    <addaction name="actionViewPanUp_" />
    <addaction name="actionViewPanDown_" />
    <addaction name="separator" />
    <addaction name="actionViewToggleShowCursorPosition_" />
    <addaction name="actionViewTogglePhantomCursor_" />
    <addaction name="actionViewToggleOutline_" />
    <addaction name="actionViewToggleToolbar_" />
   </widget>
   <addaction name="menuView_" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionViewZoomIn_" >
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/zoomIn.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
   <property name="toolTip" >
    <string>Increase magnification</string>
   </property>
   <property name="shortcut" >
    <string>I</string>
   </property>
  </action>
  <action name="actionViewZoomOut_" >
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/zoomOut.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
   <property name="toolTip" >
    <string>Reduce magnification</string>
   </property>
   <property name="shortcut" >
    <string>O</string>
   </property>
  </action>
  <action name="actionViewPanLeft_" >
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/leftArrow.png</iconset>
   </property>
   <property name="text" >
    <string>Pan Left</string>
   </property>
   <property name="shortcut" >
    <string>Left</string>
   </property>
  </action>
  <action name="actionViewPanRight_" >
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/rightArrow.png</iconset>
   </property>
   <property name="text" >
    <string>Pan Right</string>
   </property>
   <property name="shortcut" >
    <string>Right</string>
   </property>
  </action>
  <action name="actionViewPanUp_" >
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/upArrow.png</iconset>
   </property>
   <property name="text" >
    <string>Pan Up</string>
   </property>
   <property name="shortcut" >
    <string>Up</string>
   </property>
  </action>
  <action name="actionViewPanDown_" >
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/downArrow.png</iconset>
   </property>
   <property name="text" >
    <string>Pan Down</string>
   </property>
   <property name="shortcut" >
    <string>Down</string>
   </property>
  </action>
  <action name="actionViewToggleShowCursorPosition_" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/showCursorPosition.png</iconset>
   </property>
   <property name="text" >
    <string>Hide Cursor Position</string>
   </property>
  </action>
  <action name="actionViewTogglePhantomCursor_" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../guiIcons.qrc" >:/phantomCursorOn.png</iconset>
   </property>
   <property name="text" >
    <string>Hide Phantom Cursor</string>
   </property>
   <property name="shortcut" >
    <string>C</string>
   </property>
  </action>
  <action name="actionViewToggleOutline_" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Magnification Outline</string>
   </property>
   <property name="shortcut" >
    <string>CTRL+O</string>
   </property>
  </action>
  <action name="actionViewToggleToolbar_" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Toolbar</string>
   </property>
   <property name="shortcut" >
    <string>CTRL+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../guiIcons.qrc" />
 </resources>
 <connections/>
</ui>
